### ISSUE TITLE
Link Error

### ISSUE THREAD

**Issue by josephgupkim** [Ariels713/prj-rev-bwfs-dasmoto#1]
Initially I wasn't able to see the goodness from the stylesheet because you didn't close off the link.

https://github.com/Ariels713/prj-rev-bwfs-dasmoto/blob/master/Dasmoto's%20Arts%20%26%20Crafts/index.html#L6

Then I couldn't see it because you used absolute paths over relative paths.
Think about it like this:
- When I open your project up on my computer, I don't have access to "\Users\ariel\Desktop\Projects\Dasmoto's Arts & Crafts\" because I'm not Ariel and I don't have access to your Desktop, Projects, etc.
- But because I do have your "Dasmoto's Arts & Crafts" folder, you can use a path that's relative to the structure in here.
- If you switch the previously stated lines "href" value to "./resources/css/index.css"; it should work for both you and me!

**Comment by Ariels713** [comment]
Right! I copied the files path and pasted it.  Thanks for pointing that out
